src/backend/distributed/connection/remote_commands.c

```diff
@@ -277,10 +277,21 @@ ReportConnectionError(MultiConnection *connection, int elevel)
 		messageDetail = pchomp(PQerrorMessage(pgConn));
 	}
 
-	ereport(elevel, (errcode(ERRCODE_CONNECTION_FAILURE),
-					 errmsg("connection error: %s:%d", nodeName, nodePort),
-					 messageDetail != NULL ?
-					 errdetail("%s", ApplyLogRedaction(messageDetail)) : 0));
+
```

Comment:
This is not the ideal wording, and lots of tests will fail -- I'll update them once we agree on the wording / approach.

I asked review both from @marcocitus and @thanodnl as we've though with Nils that this is a good idea, but would be nice what Marco thinks.

Also,  `ApplyLogRedaction ` seems very invasive. Do we enable it on hyperscale? If so, what can we do to show the actual error message?

Reply:
I guess there might be some messages that contain the user name. Can we identify these messages somehow?

Reply:
This should probably be fixed in whatever code is producing the error message, maybe remove the ApplyLogRedaction.

Reply:
> This should probably be fixed in whatever code is producing the error message, maybe remove the ApplyLogRedaction.

Makes sense, so you're onboard with the change?

We already agreed with @thanodnl that this is useful, so I'll clean-up the PR to make it ready.

I can add a similar comment to your last comment in the code, in case readers are confused.

Reply:
Yes